app/assets/stylesheets/components/auth-form.scss

```diff
@@ -32,40 +32,40 @@
 
   .lbl {
     font-size: 14px;
-    margin: 0px;
-    line-height: 18px;
     font-weight: bold;
+    line-height: 18px;
+    margin: 0px;
   }
 
   .field {
     border: 1px solid #dae1e3;
-    font-size: 18px;
-    font-weight: 100;
-    padding: 10px 10px 10px 35px;
     border-radius: 4px;
     color: #4b5b62;
     display: block;
+    font-family: inherit;
+    font-size: 18px;
+    font-weight: 100;
     height: 44px;
+    padding: 10px 10px 10px 35px;
     width: 100%;
-    font-family: inherit;
   }
 }
 
 .button {
   border-radius: 5px;
+  box-shadow: 0 1px 0 rgba(0,0,0,.12);
+  color: #fff;
+  fill: #fff;
   font-size: 16px;
   font-weight: 400;
   height: 40px;
   line-height: 40px;
   overflow: hidden;
   padding: 0 12px;
   text-align: center;
-  -webkit-box-shadow: 0 1px 0 rgba(0,0,0,.12);
-  box-shadow: 0 1px 0 rgba(0,0,0,.12);
-  color: #fff;
-  fill: #fff;
   text-shadow: 0 -1px 0 rgba(0,0,0,.1);
   transition: none !important;
+  -webkit-box-shadow: 0 1px 0 rgba(0,0,0,.12);
```

Comment:
Avoid vendor prefixes.<br>Color literals like `rgba(0, 0, 0, 0.12)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Commas in function arguments should be followed by one space<br>`.12` should be written with a leading zero as `0.12`